clap_mangen/src/render.rs

```diff
@@ -29,10 +29,8 @@ pub(crate) fn description(roff: &mut Roff, cmd: &clap::Command) {
     }
 }
 
-pub(crate) fn synopsis(roff: &mut Roff, cmd: &clap::Command) {
-    let name = cmd.get_bin_name().unwrap_or_else(|| cmd.get_name());
+pub(crate) fn command_with_args(cmd: &clap::Command, name: String) -> Vec<Inline> {
```

Comment:
For consistency, shouldn't this be called `usage`?

Reply:
Done